content/rest-api/index.textile

```diff
@@ -1004,6 +1004,7 @@ h5. Parameters
 - channel := Filter to restrict to subscriptions associated with that @channel@.
 - clientId := Filter to restrict to subscriptions associated with that @clientId@. Cannot be used with @deviceId@.
```

Comment:
Might be clearer if we say "Do not specify both @deviceId@ and @clientId@, unless @concatFilters@ is set to true."

Reply:
I have done something slightly different, but mostly takes care of this concern:  d152b7835730411d29d9c5f9f5ba7345e6084c05

The main annoyance here is the API itself though. If `clientId` and `deviceId` are set, it should return subscriptions for both, not throw an error. Then `concatFilters` can be removed. More of my concerns about this API is https://ably-real-time.slack.com/archives/CURL4U2FP/p1630998668011900